co-log/co-log.cabal

```diff
@@ -27,18 +27,18 @@ library
                            Colog.Message
                            Colog.Monad
                            Colog.Pure
-                       Prelude
+  other-modules:       Prelude
```

Comment:
Could you update version of the package to `0.1.0` in `co-log.cabal` file and also add entry to CHANGELOG? Again, we're going to wait for more contributions before publishing version with major breaking change.